local-cluster/tests/local_cluster.rs

```diff
@@ -2002,6 +2020,39 @@ fn test_faulty_node(faulty_node_type: BroadcastStageType) {
     cluster.check_for_new_roots(16, &"test_faulty_node");
 }
 
+#[test]
+#[serial]
+#[allow(unused_attributes)]
+#[ignore]
+// Run two validators with the same keys
+fn test_duplicate_node() {
```

Comment:
I think we can remove this test since running two validators with the same keys causes an autokill once they detect each other in gossip now

Reply:
couldn't a malicious user disable the autokill?

Reply:
yeah, but since we don't disable that, this will kill the nodes and then the test will fail right?

Reply:
Ah sure, removed! I'll look into adding that back in a follow up PR